staging/src/k8s.io/client-go/kubernetes/clientset.go

```diff
@@ -457,10 +457,6 @@ func NewForConfig(c *rest.Config) (*Clientset, error) {
 	if err != nil {
 		return nil, err
 	}
-	cs.batchV1beta1, err = batchv1beta1.NewForConfig(&configShallowCopy)
```

Comment:
That's not necessary, we still want to wire these here.

Reply:
Thanks. I will change it.